packages/terra-time-input/src/terra-dev-site/doc/example/TimeInputSeconds.jsx

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 // eslint-disable-next-line import/no-unresolved, import/extensions
 import TimeInput from 'terra-time-input/lib/TimeInput';
+import classNames from 'classnames/bind';
+import styles from './TimeInput.module.scss';
```

Comment:
I'd make a new stylesheet thats used exclusively for the example, rather than reuse the TimeInput's stylesheet. Consumers could get the wrong idea if the review the example source.

Reply:
My bad, I had created a new stylesheet for the example but named it the same as TimeInput's stylesheet.
I have now renamed the new stylesheet to be more appropriate.